### ISSUE TITLE
Stockpile import from shopping list

### ISSUE THREAD

**Issue by eve-n0rman** [GoldenGnu/jeveassets#16]
It'd be nice to have a multi-buy style shopping list import function for the Stockpile feature.

**Comment by GoldenGnu** [comment]
Could you explain this feature suggestion in greater detail?
I'm not quite sure I understand what you want to do, sorry...

**Comment by eve-n0rman** [comment]
I’d like to be able create a stockpile from a list of items and quantities, as can be used to purchase things on the market using multi-buy or as you can currently *export* from stockpiles. Today I can only import a stockpile from EFT, EVE IPH, or XML.

**Comment by GoldenGnu** [comment]
Thank you for explaining the feature for me :) 

It's 2 different formats, but, eve understand the jEveAssets shopping list format when importing (which is nice ofc.).

Eve multi-buy export:
```
Iridium Charge S	100	8,17	817,00
Inferno Rocket	1	6,64	6,64
Total:			823,64
```

jEveAssets shopping list:
```
1x Inferno Rocket
100x Iridium Charge S
```

Do you need to be able to import both formats?

**Comment by eve-n0rman** [comment]
Both would be handy.  The EVE multi-buy import is actually pretty flexible, so your jEveAssets shopping list format is probably sufficient as a starter.

**Comment by GoldenGnu** [comment]
Thanks for the answer.
It will be some time before I have time to work on anything else than ESI.
I will get back to do feature suggestions eventually.

**Comment by GoldenGnu** [comment]
Released in 5.4.0
